integrations/GATEWAYS_FILE.md

```diff
@@ -0,0 +1,91 @@
+# IPIP 0280: Gateways file
+- Start Date: 2022-12-08
+- Related Issues:
+  - https://github.com/ipfs/kubo/issues/8847
+
+## Summary
+
+Defines how IPFS implementations must expose their gateway. This only affects the file-based method of exposiing a gateway.
+
+## Motivation
+
+Applications wanting to use IPFS resources are, without this spec, left to invent their own ways of finding a gateway. This spec defines how an application wanting to implement IPFS support can find a list of gateways.
+
+Users should be aware that this is just a list of gateways. It could contain a local gateway but doesn't have to. It's up to the user to do filtering on that list to find an actual local gateway.
+
+## Detailed design
+
+This integration spec defines the recommended way for IPFS implementations to expose the gateway they host for IPFS integrations (think applications wanting to support IPFS) to find and use said gateways.
+
+An IPFS implementation must expose the gateway it serves in a file called `gateways` (more on where this file is located in the section below). The gateway must be exposed as a single line in the following format:
```

Comment:
> more on where this file is located in the section below
Should be an anchor jumping link to `#Gateways file placement`

#Gateways file placement